classes/class3/catscatscats/views/allCats.hbs

```diff
@@ -0,0 +1,6 @@
+<h1>You want a look at every cats!</h1>
+<ul>
+{{#each cats}}
+	<li>{{this}}</li>
```

Comment:
Ideally you would break out more specific information from the cat object, rather than just displaying a string.